src/libraries/Common/src/Interop/Windows/Kernel32/Interop.Globalization.cs

```diff
@@ -40,6 +40,11 @@ internal static unsafe partial class Kernel32
 
         internal const uint TIME_NOSECONDS = 0x00000002;
 
+        internal const int GEOCLASS_NATION       = 16;
+        internal const int GEO_ISO2              =  4;
+        internal const int GEOID_NOT_AVAILABLE   = -1;
+        internal const int EN_LANG_ID            =  9;
```

Comment:
Should this be `LANG_ENGLISH` ? I do not see `EN_LANG_ID` defined in the Windows SDK.

Reply:
I'll change this too.